### PR TITLE
Fix build issues and update to sling 47

### PR DESCRIPTION
**Pull request by apelluru**

### CODE REVIEW

**Review by rombert — CHANGES_REQUESTED** [review]
Thanks for the PR @ashokmca07 ! I see the build completes successfully with Java 11 now, which is great. One thing which still needs attention is the list of exported packages.

Before your PR, this was present in the generated manifest:

```
Export-Package: org.apache.sling.commons.cache.api;version="1.0"
```

Now it's listed as a private package.

```
Private-Package: org.apache.sling.commons.cache.api
```

Can you please fix this?

**Comment by rombert** [comment]
Sorry, this was for the API PR, please ignore for now.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 
-# Apache Sling Cache API Implementation support library
+# Apache Sling Cache API Implementation library
```
Comment (rombert) [inline README.md]: Not sure this is needed.

File: pom.xml [hunk 2]
```diff
@@ -22,26 +22,31 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.sling</groupId>
-        <artifactId>sling</artifactId>
-        <version>16</version>
-        <relativePath/>
+        <artifactId>sling-bundle-parent</artifactId>
```
Comment (rombert) [inline pom.xml]: This is not an OSGi bundle, so please use org.apache.sling:sling:47 instead.

File: pom.xml [hunk 3]
```diff
@@ -22,26 +22,31 @@
...
 
     <name>Apache Sling Cache API Implementation support library</name>
     <description>
         This bundle provides an implementation support for Cache implementations. It is a utility jar that should be embedded.
     </description>
 
+    <properties>
+        <sling.java.version>8</sling.java.version>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
+        <bnd.baseline.skip>true</bnd.baseline.skip>
```
Comment (rombert) [inline pom.xml]: This should not be needed.

**Review by rombert — CHANGES_REQUESTED** [review]
Thanks for the PR @ashokmca07 ! I added a couple of comments, please review.

File: README.md (hunk 1 above)
Comment (apelluru) [reply README.md]: Sorry it was my mistake. reverted

File: pom.xml (hunk 2 above)
Comment (apelluru) [reply pom.xml]: Fixed

File: pom.xml (hunk 3 above)
Comment (apelluru) [reply pom.xml]: Removed

**Comment by apelluru** [comment]
Thanks for letting me know @rombert about https://github.com/apache/sling-org-apache-sling-commons-cache-impl/pull/1#pullrequestreview-927248093

will fix it in PR for org.apache.sling.commons.cache.api

**Comment by rombert** [comment]
Thanks for the quick turnaround @ashokmca07 , I have merged this PR.

**Pull request merged**